egs/aishell/asr/simple_v1/train.py

```diff
@@ -260,8 +260,7 @@ def main():
     graph_compiler = CtcTrainingGraphCompiler(
         L_inv=L_inv,
         phones=phone_symbol_table,
-        words=word_symbol_table,
-        oov='<SPOKEN_NOISE>'
+        words=word_symbol_table
     )
     phone_ids = get_phone_symbols(phone_symbol_table)
```

Comment:
`get_phone_symbols` is not imported in this file.
Please have a look at 
https://github.com/k2-fsa/snowfall/blob/322fbe8f788ee8eedd15c781f0821df46fdc84b7/egs/librispeech/asr/simple_v1/ctc_train.py#L30

and please update `decode.py` accordingly.

---

The training set is not using MUSAN for data augmentation. Please refer to
https://github.com/k2-fsa/snowfall/blob/322fbe8f788ee8eedd15c781f0821df46fdc84b7/egs/librispeech/asr/simple_v1/ctc_train.py#L274-L275

https://github.com/k2-fsa/snowfall/blob/322fbe8f788ee8eedd15c781f0821df46fdc84b7/egs/librispeech/asr/simple_v1/ctc_train.py#L281-L283

Reply:
Got it. I will update with latest librispeech recipe and run a test.